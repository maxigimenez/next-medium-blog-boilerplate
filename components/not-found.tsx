import React from 'react';
import Head from 'next/head';
import config from '@/config';

export const NotFound = () => (
  <>
    <Head>
      <title>{config.title}</title>
    </Head>
    <div className="d-flex justify-content-center align-items-center" style={{ height: '100vh' }}>
      <h1 className="mr-3 pr-3 align-top border-right inline-block align-content-center">404</h1>
      <div className="inline-block align-middle">
        <h2 className="font-weight-normal lead" id="desc">
          The page you requested was not found.
        </h2>
      </div>
    </div>
  </>
);
