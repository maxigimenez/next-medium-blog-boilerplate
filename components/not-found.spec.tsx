import React from 'react';
import { shallow } from 'enzyme';
import { NotFound } from './not-found';

describe('not-found', () => {
  describe('snapshots', () => {
    test('render default', () => {
      const component = shallow(<NotFound />);
      expect(component).toMatchSnapshot();
    });
  });
});
