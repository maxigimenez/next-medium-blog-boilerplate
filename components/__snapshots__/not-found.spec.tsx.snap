// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not-found snapshots render default 1`] = `
<Fragment>
  <Head>
    <title>
      Demo Blog - Next.js medium style boilerplate
    </title>
  </Head>
  <div
    className="d-flex justify-content-center align-items-center"
    style={
      Object {
        "height": "100vh",
      }
    }
  >
    <h1
      className="mr-3 pr-3 align-top border-right inline-block align-content-center"
    >
      404
    </h1>
    <div
      className="inline-block align-middle"
    >
      <h2
        className="font-weight-normal lead"
        id="desc"
      >
        The page you requested was not found.
      </h2>
    </div>
  </div>
</Fragment>
`;
